context.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ga4gh
```

Comment:
I wonder if you are better to have a sub package called ga4gh instead of this odd renaming

or even just identity

Reply:
As discussed offline, we definitely don't want to hide the renaming so please put the names back.  It would be best if it wasn't surprising but I understand you don't want another ga4gh level in the tree.

Reply:
See my [previous suggestion](https://github.com/googlegenomics/ga4gh-identity/pull/1#discussion_r217899837) of either `getEvaluator` or `buildEvaluator`, either of which would be fine.